src/service/http-server.ts

```diff
@@ -49,6 +61,14 @@ export class HttpServer {
 
     this.app.get('/render', asyncMiddleware(this.render));
     this.app.get('/render/csv', asyncMiddleware(this.renderCSV));
+    this.app.post(
+      '/render/sanitize',
```

Comment:
```suggestion
      '/sanitize',
```
This would be better if we could get rid of the `render` prefix and do something in Grafana to replace `/render` in the renderer URL with `/sanitize` for example.
Else, it will mess up the metrics we collect about image renderer (set up here: https://github.com/grafana/grafana-image-renderer/blob/master/src/service/metrics_middleware.ts#L49). It could be interesting to have metrics for this endpoint as well but we should keep them separate.